packages/PoppyPrint-Core.package/PPFormatter.class/instance/evaluateBraceStatements.multiLine..st

```diff
@@ -0,0 +1,16 @@
+helper
+evaluateBraceStatements: aNode multiLine: multiLine
+
+	aNode elements do: [:element | | isLast |
+		isLast := element = aNode elements last.
+		((self isOptionalNewLineMarker: element) and: [multiLine]) ifFalse: [
+			((self isEmptyLineMarker: element) or: [self isOptionalEmptyLineMarker: element])
+				ifTrue: [self newLine]
+				ifFalse: [
+					(self isOptionalNewLineMarker: element) ifFalse: [
```

Comment:
could be that I'm reading this wrong, but isn't this always false because if it were true, control flow would have gone into the upper ifTrue branch?

One suggestion, to help clarify the method, could be to create a separate `printBraceElement:multiLine:` method and make use of early returns.